.gitignore

```diff
@@ -13,3 +13,5 @@ profile.coverprofile
 /gin-bin
 /.vscode
 /federation/etc/.env*
+/.act*
+/workflow
```

Comment:
I'm using https://github.com/nektos/act to test pipelines locally. This produces /workflow folder as output. I ignored /.act* type of files because you can send events in json to this simulator... $act -b -e .act-push-event.json (for instance)

Reply:
ok.. no problems with .act... 
but why do we ignore /workflow?

Reply:
running act produces workflow where it stores output of jobs

Reply:
Understood. 

_on a sidenote.... our ignore files are getting bulkier, might be good to put a comment or two on some lines, especially when they use keywords (workflow) that are used in the product itself._